src/ui/ResultLink/ResultLink.ts

```diff
@@ -284,15 +284,10 @@ export class ResultLink extends Component {
   }
   public renderUri(element: HTMLElement, result?: IQueryResult) {
     if (/^\s*$/.test(this.element.innerHTML)) {
-      if (!this.options.titleTemplate) {
-        this.element.innerHTML = this.result.title
-          ? HighlightUtils.highlightString(this.result.title, this.result.titleHighlights, null, 'coveo-highlight')
-          : this.result.clickUri;
-      } else {
-        let newTitle = StringUtils.buildStringTemplateFromResult(this.options.titleTemplate, this.result);
-        this.element.innerHTML = newTitle
-          ? StreamHighlightUtils.highlightStreamText(newTitle, this.result.termsToHighlight, this.result.phrasesToHighlight)
-          : this.result.clickUri;
+      const title = this.getDisplayedTitle(result);
+      this.element.innerHTML = title;
+      if (!this.element.title) {
```

Comment:
You are checking so a user could've already added a title, is that it?

Also looking at the doc, there's a `?` for IE, have you checked it works correctly in IE11? https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/title

Thanks! 🚀

Reply:
Yes, it sounds like it would be annoying for a user to set a custom title only for it to be overwritten unexpectedly after an update.

Hmm, I haven't checked IE11, I'll check it out.

Reply:
UPDATE: I tested, and indeed it seems to work with IE11.